src/appleseed.shared/application/progresstilecallback.cpp

```diff
@@ -80,12 +81,19 @@ namespace
 
         void on_tiled_frame_begin(const Frame* frame) override
         {
-            m_stopwatch.start();
+            // Do not restart the stopwatch if it is already running.
+            if (m_stopwatch.get_seconds() <= 0.0)
```

Comment:
That's not robust. Maybe the stopwatch just started and at this point 0.0 seconds have elapsed, this could happen with low resolution timers.

Reply:
In that case, I think a check based on `m_rendered_pixels` or `m_rendered_tiles` would be more robust.